test/view/example_test.js

```diff
@@ -0,0 +1,27 @@
+var App = require('./../../app/example'),
+    React = require('react');
+
+function getNewMountPoint() {
+    var domNode = document.createElement('div');
+    document.body.appendChild(domNode);
+    return domNode;
```

Comment:
If you wanna be clever about it, you could export this function and the renderComponent function into a lib of your own that you then pull in to each view level unit test.